### PR TITLE
[chore] add "Seeking codeowners" to Windows Performance Counters receiver

### PR DESCRIPTION
**Pull request by andrzej-stencel**
Based on a [message](https://cloud-native.slack.com/archives/C01N6P7KR6W/p1713448144169939?thread_ts=1713439927.384819&cid=C01N6P7KR6W) from @dashpole on CNCF Slack.

> We should probably find a new owner for the windows perf counter receiver.

### CODE REVIEW

**Review by songy23 — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]

**Comment by alxbl** [comment]
As discussed on Slack, I have sought (and received) approval from my employment to pursue code ownership for the component. Assuming there is still a need for code owners, I would be willing to contribute some of my time to become one.

I am not a member of the OpenTelemetry org yet, and I will be following the process [outlined in the doc](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#requirements).

If there is anything else that I should do/be doing, please let me know.

Thanks,

**Review by atoulme — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Comment by codeboten** [comment]
Thanks @alxbl! Becoming a member and then opening a PR to add yourself as a code owner would be great!

**Pull request merged**
